devenv/dev-dashboards-without-uid/panel_tests_polystat.json

```diff
@@ -448,6 +448,10 @@
             {
               "text": "Malaysian Ringgit (RM)",
               "value": "currencyMYR"
+            },
+	    {
```

Comment:
Can we fix the format (some spaces are missing)? 🙈

Reply:
Damned, $EDITOR put a tab when I fixed conflicts!  Commit amended, sorry for the bore.

Reply:
no worries 😄